zhmcclient/_lpar.py

```diff
@@ -1535,9 +1536,19 @@ def reset_clear(self, force=False, wait_for_completion=True,
           :exc:`~zhmcclient.ConnectionError`
           :exc:`~zhmcclient.OperationTimeout`: The timeout expired while
             waiting for completion of the operation.
-          :exc:`~zhmcclient.StatusTimeout`: The timeout expired while
-            waiting for the desired LPAR status.
         """
+        if status_timeout is not None:
```

Comment:
Same check as above, couldn't that go into a single helper method?

Reply:
Moved into a helper method `warn_deprecated_parameter()` in the utils module.